contracts/contracts/ramps/wise/WiseRamp.sol

```diff
@@ -0,0 +1,856 @@
+//SPDX-License-Identifier: MIT
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+import { Bytes32ArrayUtils } from "../../external/Bytes32ArrayUtils.sol";
+import { Uint256ArrayUtils } from "../../external/Uint256ArrayUtils.sol";
+
+import { ITLSData } from "./interfaces/ITLSData.sol";
+import { IWiseRegistrationProcessor } from "./interfaces/IWiseRegistrationProcessor.sol";
+import { IWiseSendProcessor } from "./interfaces/IWiseSendProcessor.sol";
+
+pragma solidity ^0.8.18;
+
+contract WiseRamp is Ownable {
+
+    using Bytes32ArrayUtils for bytes32[];
+    using Uint256ArrayUtils for uint256[];
+
+    /* ============ Events ============ */
+    event AccountRegistered(address indexed accountOwner, bytes32 indexed onRampId, bytes32 indexed wiseTagHash);
```

Comment:
Think we should still call onRampId as just id. because it would be held by everyone in the system, on-rampers and off-rampers alike. That is true for wise. Although not sure how that would scale beyond wise.

Reply:
Yeah one of the open questions I've been considering too. I'll make that change